### ISSUE TITLE
Performance 

### ISSUE THREAD

**Issue by qianwangn** [kkhoot/PAA#9]
I implement PAA based on detectron2.
But  only got 40AP without score voting, with 1x setting.
does any reasons that cost 0.5 points lower?

**Comment by kai-xl8** [comment]
Hi, thanks for your work! I don't know the exact reason because I didn't see your code, but the reason might be in the separation boundary. In the paper we reported that All boundaries (Fig. 3 (a) ~ (b)) show similar performance, but after repetitive experiments we found that Fig. 3 (c) is the most stable one. So I recommend you to try Fig. 3 (c). We will update our paper accordingly. Thanks again for your work!

**Comment by kkhoot** [comment]
Hi, thanks for your work! I don't know the exact reason because I didn't see your code, but the reason might be in the separation boundary. In the paper we reported that All boundaries (Fig. 3 (a) ~ (b)) show similar performance, but after repetitive experiments we found that Fig. 3 (c) is the most stable one. So I recommend you to try Fig. 3 (c). We will update our paper accordingly. Thanks again for your work!
